dependencies.yaml

```
@@ -304,7 +304,7 @@ dependencies:
           - nbsphinx
           - numba>=0.59.1,<0.61.0a0
           - numpydoc
-          - sphinx
+          - sphinx<8.2.0
```

```suggestion
          # this pin can be reverted when https://github.com/spatialaudio/nbsphinx/issues/825 is resolved
          - sphinx<8.2.0
```

Can we please leave a comment here explaining why the pin's there and when it can be removed?